packages/angular/src/generators/convert-to-buildable/generators/add-library-build-target/add-library-build-target.ts

```diff
@@ -0,0 +1,36 @@
+import { readProjectConfiguration, Tree, updateProjectConfiguration } from '@nrwl/devkit';
+import * as path from 'path';
+
+import { NormalizedSchema } from '../../util';
+
+export function addLibraryBuildTarget(
+  host: Tree,
+  { projectName, projectRoot }: NormalizedSchema
+): void {
+  const project = readProjectConfiguration(host, projectName);
+
+  if (project.targets.build) {
+    console.warn(
+      `Project with name "${projectName} already has a "build" target. Overwriting..."`
+    );
+  }
```

Comment:
Maybe we should have a `--force` option to override existing targets and otherwise leave them as is.